out.csv

```diff
@@ -0,0 +1,2 @@
+Name,Email,Company,Position,Able to Give Referrals?,Resume
```

Comment:
Let's try to put this in gitignore? also do we want this file saved or just sent back? thoughts?